build-system/common/travis.js

```diff
@@ -48,131 +48,98 @@ function isTravisPushBuild() {
   return isTravisBuild() && process.env.TRAVIS_EVENT_TYPE === 'push';
 }
 
+/**
+ * Returns a function to test the environment and look up a Travis environment
+ * variable
+ * @param {function():boolean} testFn
+ * @param {string} errorMsg
+ * @param {string} envKey
+ * @return {function():string}
+ */
+function travisEnv(testFn, errorMsg, envKey) {
+  return function () {
+    if (!testFn()) {
+      log(red('ERROR:'), errorMsg, 'Cannot get', cyan(`process.env.${envKey}`));
+    }
+    return process.env[envKey];
+  };
+}
+
+const travisBuildEnv = (envKey) =>
+  travisEnv(isTravisBuild, 'This is not a Travis build.', envKey);
+
 /**
  * Returns the build number of the ongoing Travis build.
  * @return {string}
  */
-function travisBuildNumber() {
-  if (!isTravisBuild()) {
-    log(
-      red('ERROR:'),
-      'This is not a Travis build. Cannot get',
-      cyan('process.env.TRAVIS_BUILD_NUMBER') + '.'
-    );
-  }
-  return process.env.TRAVIS_BUILD_NUMBER;
-}
+const travisBuildNumber = travisBuildEnv('TRAVIS_BUILD_NUMBER');
```

Comment:
With this refactor, `travisBuildNumber`, etc. have all changed from a function to a constant, and should no longer have function-style JSDoc (with the `@return`, etc.).

(The same thing applies to all the `timedExec*` constants (formerly functions) in `exec.js` that were refactored in #30118.)

Reply:
My bad, I saw this PR turn green and I didn't see your comment until after the merge. Sorry bout that! Fixing up with `@function` annotation now

Reply:
(GitHub AJAX updates don't hit all parts of the UI, annoyingly)

Reply:
https://github.com/ampproject/amphtml/pull/30556